docs/metrics.md

```diff
@@ -0,0 +1,120 @@
+# Metrics
+
+Test metrics are sent to 3 different places:
+1. reported properies
+2. test telemetry message bodies
+3. Azure Monitor
+
+Not all metrics are sent to all destinations.
+
+| group | reported properties | test telemetry | Azure Monitor |
+| Session Metrics | X | X | - |
```

Comment:
Why not send session metrics to azure monitor?

Reply:
This structure has run start time, run end time, exit reason, last update time, and paring id.  These are either static or redundant.